PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -3982,11 +3977,7 @@ Bool_t AliConvEventCuts::MimicTrigger(AliVEvent *event, Bool_t isMC ){
       Double_t threshold = thresholdEMCalL1[binRun];
 
       if (isMC && spreadEMCalL1[binRun] != 0.){
-        TF1 triggerSmearing("triggerSmearing","[0]*exp(-0.5*((x-[1])/[2])**2)",0,15);
-        triggerSmearing.SetParameter(0, 1/(spreadEMCalL1[binRun]*TMath::Sqrt(TMath::Pi()*2)));
-        triggerSmearing.SetParameter(1, thresholdEMCalL1[binRun]);
-        triggerSmearing.SetParameter(2, spreadEMCalL1[binRun]);
-        threshold = triggerSmearing.GetRandom();
+        threshold = fRandom.Gaus(thresholdEMCalL0[binRun], spreadEMCalL0[binRun]);
```

Comment:
copy/paste error: L1 here!

Reply:
Sorry that was a stupid mistake... Thanks for checking! It should be corrected now for both